src/module.c

```diff
@@ -1052,7 +1064,7 @@ int RediSearch_InitModuleInternal(RedisModuleCtx *ctx, RedisModuleString **argv,
 
   RM_TRY(RedisModule_CreateCommand, ctx, RS_CONFIG, ConfigCommand, "readonly", 1, 1, 1);
 
-#ifndef RS_COORDINATOR
+#ifdef FORCE_CROS_SLOT_VALIDATION
   // we are running in a normal mode so we should raise cross slot error on alias commands
   RM_TRY(RedisModule_CreateCommand, ctx, RS_ALIASADD, AliasAddCommand, "readonly", 1, 2, 1);
   RM_TRY(RedisModule_CreateCommand, ctx, RS_ALIASUPDATE, AliasUpdateCommand, "readonly", 1, 2, 1);
```

Comment:
Should we use another define here? Can eliminate Lines 1075/6

Reply:
This is how it was before so I decided to stick to it, but yeah sure ...